package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "test": "mocha --UV_THREADPOOL_SIZE=100 test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
+    "pretest": "source ./setup.sh",
```

Comment:
I'm not sure if we need to have the `setup.sh` as the pretest, because it will imply that whoever running the tests need to use the oracle instance in docker.

Reply:
How do we setup Oracle for CI?

Reply:
@rmg, do you know?

Reply:
@rmg apart from an Oracle instance, we have to populate some data in it. I got `setup.sh` to do it's work (pulling the desired Oracle image, running it, and populating data), but `ORACLE_USER` and `ORACLE_PASSWORD` are not available to the app, despite `export ORACLE_USER=SYSTEM ORACLE_PASSWORD=oracle && mocha --UV_THREADPOOL_SIZE=100 test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js`.

On the "Environment variables" page of the project on Jenkins, there's no entry for `ORACLE_USER` and `ORACLE_PASSWORD`, while it is there for Cloudant, DashDB, Redis etc.

Reply:
If the static instance already has the data populated, we will not need to run `setup.sh`, just need `ORACLE_USER` and `ORACLE_PASSWORD` set in the  "Environment variables" for the project.

Reply:
The credentials are already provided via `rc`. The tests all have a line like this in them:
```js
const config = require('rc')('loopback', {test: {oracle: {}}}).test.oracle;
```

That is pulling values from `~/.loopbackrc`, which is being injected in the test environment specifically to provide those credentials. The instance should already contain the test data as far as I'm aware. The purpose of `setup.sh` was to create a local DB that matches that environment.